frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -324,6 +324,9 @@ export class FeedItemWithPreview extends React.Component {
         ? this.getFirstComment()
         : null
     )
+    if (shouldShowComment && props.inRecentActivities) {
+      props.content.firstComment = commentToShow
```

Comment:
You must not do that
Never programmatically affect a props in the component. Props are meant to be set by parents, reaffecting it in the component will lead to unexpected behavior
@Millefeuille42 @gbonaspetti